Demo/src/main/java/com/paypal/android/ui/card/CardFields.kt

```diff
@@ -14,18 +15,31 @@ import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.TextRange
 import androidx.compose.ui.text.input.KeyboardType
 import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.unit.dp
 import com.paypal.android.R
 
 @Composable
 fun CardFields(
     cardViewModel: CardViewModel,
     modifier: Modifier = Modifier
 ) {
+    val orderID: String by cardViewModel.orderID.observeAsState("")
```

Comment:
Is this just added until we get the Demo app networking PR up?  If we want this demo app to be similar to a merchant app, having this as an input field doesn't seem right to me - can we have somewhere in the code where we can update this hard-coded value until the networking PR is complete?

Reply:
Ok cool I can revert.